### PR TITLE
Giving Tuesday 2024

### PR DESCRIPTION
**Pull request by ashley-hebler**
#### What's this PR do?

Adds some updates for Giving Tuesday that falls on 12/2

#### Why are we doing this? How does it help us?

Membership drive

#### How should this be manually tested?

Review on https://donations-testing.herokuapp.com/donate
Just a general check on design, broken HTML etc. Emily is reviewing that this matches her spec separately

Review on https://waco-donations-testing-1119390b0075.herokuapp.com/donate
Confirm that nothing is changed here. I didn't notice that it disrupted anything on the waco site but let me know if I overlooked

#### How should this change be communicated to end users?

Deploy on 12/2 and let Emily know

#### Are there any smells or added technical debt to note?

Sounds like some of this might just be default independent of the drive. I tried to make it easy to undo with USE_THERMOMETER=False, but we'll see how much of that applies post-drive

#### What are the relevant tickets?

https://airtable.com/appyo1zuQd8f4hBVx/pagaqtKlLeA1VeYTv?detail=eyJwYWdlSWQiOiJwYWdsUzN3Y2tEdHB3elF6UCIsInJvd0lkIjoicmVjaHV2UWpHS1Y2cVBKcHciLCJzaG93Q29tbWVudHMiOmZhbHNlLCJxdWVyeU9yaWdpbkhpbnQiOm51bGx9

#### Have you done the following, if applicable:
***(optional: add explanation between parentheses)***

* [ ] Added automated tests? *( )*
* [ ] Tested manually on mobile? *( )*
* [ ] Checked BrowserStack? *( )*
* [ ] Checked for performance implications? *( )*
* [ ] Checked accessibility? *( )*
* [ ] Checked for security implications? *( )*
* [ ] Updated the documentation/wiki? *( )*

#### TODOs / next steps:

* [x] *Get social share graphic*
* [x] *Have Emily I review*

### CODE REVIEW

File: server/templates/donate-form.html [hunk 1]
```diff
@@ -57,17 +59,29 @@ <h2 class="grid_separator link--teal">Become a Texas Tribune Member</h2>
...
+            {% include 'includes/faq.html' %}
+          </div>
+          <div class="col_7">
+            {% include 'includes/benefits.html' %}
+          </div>
+        </div>
+        {% else %}
+          <div class="member_benefits grid_container">
+            {% include 'includes/benefits.html' %}
+          </div>
```
Comment (matthewdylan) [inline server/templates/donate-form.html:83]: **Nitpick (non-blocking):** annoying again, but can you fix this indentation?

File: server/templates/donate-form.html [hunk 2]
```diff
@@ -35,16 +31,22 @@ <h1 class="splash__headline">
   </section>
 
   <main class="main">
+    {% if use_thermometer %}
+    {% include "includes/thermometer.html" %}
+  {% endif %}
```
Comment (matthewdylan) [inline server/templates/donate-form.html:36]: **Nitpick (non-blocking):** I'm being so annoying but can you fix the indentation here

**Review by matthewdylan — APPROVED** [review]
LGTM and works great. Have the most nitpick of nitpicks on indentation in two places but feel free to say pishposh to all that.

File: server/templates/donate-form.html (hunk 1 above)
Comment (ashley-hebler) [reply server/templates/donate-form.html]: Fixed in [5fd0d92](https://github.com/texastribune/donations/pull/1231/commits/5fd0d927f22755b9e70d24cd5916b0b7ae2ff404), good catch!

File: server/templates/donate-form.html (hunk 2 above)
Comment (ashley-hebler) [reply server/templates/donate-form.html]: Also fixed in [5fd0d92](https://github.com/texastribune/donations/pull/1231/commits/5fd0d927f22755b9e70d24cd5916b0b7ae2ff404)

**Pull request merged**